includes/tracks/class-wc-site-tracking.php

```diff
@@ -30,6 +30,63 @@ public static function tracks_product_updated( $product_id, $post ) {
 		WC_Tracks::record_event( 'update_product', $properties );
 	}
 
+	/**
+	 * Route product importer action to the right callback.
+	 *
+	 * @return void
+	 */
+	public static function tracks_product_importer() {
```

Comment:
Probably good idea to start slice this file and start create new classes for each kind of event.
Since now we are tracking product updated and product importer will be a good start.

Reply:
Can you elaborate on the separation you'd like to see? Do you want a separate class for every individual event, or are you thinking we'd group events into "categories" like "product", "importer", "coupon", etc?

Reply:
@jeffstieler into categories sounds good, but anything to avoid classes like `WC_Ajax` or `WC_Form_Handler` which both handle multiple kinds of data and are very big classes.

Reply:
Done in 2426a25.